array_analyzer/workflows/registration_workflow.py

```diff
@@ -225,14 +223,14 @@ def point_registration(input_folder, output_folder, debug=False):
 
             # Save mask of the well, cropped grayscale image, cropped spot segmentation.
             io.imsave(output_name + "_well_mask.png",
-                      (255 * placed_spotmask).astype('uint8'))
+                      (np.iinfo(placed_spotmask.dtype).max * placed_spotmask).astype('uint8'))
```

Comment:
Actually here you still need to multiply with 255 otherwise you'll risk saving a binary image, e.g. if spotmask is a normalized float.

Reply:
oops yes.  I misinterpreted your "requested changes".  Will change these back.  Thanks!